sympy/solvers/tests/test_solveset.py

```diff
@@ -931,3 +932,15 @@ def test_issue_9557():
 
     assert solveset(x**2 + a, x, S.Reals) == Intersection(S.Reals,
         FiniteSet(-sqrt(-a), sqrt(-a)))
+
+
+def test_issue_9778():
+    assert solveset(x**3 + 1, x, S.Reals) == FiniteSet(-1)
+    assert solveset(x**(S(3)/5) + 1, x, S.Reals) == S.EmptySet
```

Comment:
I guess this should be an `EmptySet`
Since

Supposing that you must be thinking of `-1` to be a solution of this, you can check for the value
```
>>> expr = x**(S(3)/5) + 1
>>> expr.subs({x: -1})
1 + (-1)**(3/5)   # which is non-zero
```

Also [death note](http://www.wolframalpha.com/input/?i=solve%28x**%283%2F5%29+%2B+1%29) gives the same solution).

Mathematically it is different from finding the `real-value` of `(-1)**(5/3)`. Though personally i always find it quite `interesting` that.